cluster/up.sh

```diff
@@ -25,15 +25,23 @@ if [[ "$KUBEVIRT_PROVIDER" != external ]]; then
     $(kubevirtci::path)/cluster-up/up.sh
 fi
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
-
-# wait for cluster operator
-./cluster/kubectl.sh wait networkaddonsconfig cluster --for condition=Available --timeout=800s
-
+if [[ "$CNAO_PROVIDER" != external ]]; then
+  # Deploy CNAO
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/namespace.yaml
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/network-addons-config.crd.yaml
+  ./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${CNAO_VERSIOV}/operator.yaml
+  # Deploy NetworkAddonsConfig
+  ./cluster/kubectl.sh apply -f ./hack/cna/cna-cr.yaml
+  # wait for cluster operator
+  ./cluster/kubectl.sh wait networkaddonsconfig cluster --for condition=Available --timeout=800s
+else # CNAO_PROVIDER == external
```

Comment:
You added a new attribute for users to use. Document it. And follow https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool/master/.github/PULL_REQUEST_TEMPLATE.md next time you open a PR

Reply:
ack